notebooks/correlogram.ipynb

```diff
@@ -0,0 +1,893 @@
+{
```

Comment:
Going to piggyback on [Martin's comment](https://github.com/pysal/esda/pull/253/files#r1283745492) and request a title a brief explanation be added in the notebook.